src/vma/dev/ring_simple.cpp

```diff
@@ -891,29 +1034,24 @@ inline void ring_simple::vma_poll_process_recv_buffer(mem_buf_desc_t* p_rx_wc_bu
 	p_rx_wc_buf_desc->path.rx.vma_polled = false;
 }
 
-
-
-
-// All CQ wce come here for some basic sanity checks and then are distributed to the correct ring handler
-// Return values: false = Reuse this data buffer & mem_buf_desc
-bool ring_simple::rx_process_buffer(mem_buf_desc_t* p_rx_wc_buf_desc, transport_type_t transport_type, void* pv_fd_ready_array /*=NULL*/)
+#if defined(FLOW_TAG_ENABLE)
+inline void ring_simple::vma_poll_process_recv_buffer_ft(mem_buf_desc_t* p_rx_wc_buf_desc)
 {
-	size_t sz_data = 0;
+	//size_t sz_data = 0;
```

Comment:
why moved from here? not sure it is allowed to have commented code in production.